### PR TITLE
[EBPF] renamed nvmlmetrics pkg to nvidia

### PR DESCRIPTION
**Pull request by val06**
### What does this PR do?

renamed `nvmlmetrics` pkg to `nvidia` 

### Motivation

in the future we will have separate packages per gpu vendor to retrieve native device metrics via API

### Additional Notes

[Jira ticket](https://datadoghq.atlassian.net/browse/EBPF-597)

### CODE REVIEW

File: pkg/gpu/context.go [hunk 1]
```diff
@@ -46,20 +46,23 @@ func getSystemContext(nvmlLib nvml.Interface) (*systemContext, error) {
 	return ctx, nil
 }
 
-func (ctx *systemContext) queryDevices() error {
```
Comment (gjulianm) [inline pkg/gpu/context.go:49]: I wouldn't rename this function, as in other PRs we'll need to get more data for each device (e.g., the compute architecture version) in this function.

Comment (val06) [reply pkg/gpu/context.go:49]: changed to `fillDeviceInfo`

**Review by gjulianm — APPROVED** [review]

**Pull request merged**